recipes/paratext/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "0.1.1" %}
+
+package:
+  name: paratext
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/wiseio/paratext.git
+  git_rev: 2f2824eb8c8f1d060c1105531bc7ab9c44b55246
+
+build:
+  number: 1
+  skip: True  # [win or osx or py3k]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - swig
+    - pandas
+  run:
+    - python
+    - pandas
+
+test:
+  imports:
+    - paratext
```

Comment:
Given that this may have a bit of a complicated build is there any short test we can run (e.g. `run_test.py`) to check that it can actually read files?.